### DISCUSSION TITLE
Check all, uncheck all for checkbox interface with json type

### DISCUSSION THREAD

**Discussion by hermesalvesbr** [directus/directus#7511] [Feature Requests]
![uncheck](https://user-images.githubusercontent.com/379850/130141639-614e10f2-61bc-44c4-b566-a0b82526e510.png)
When we use the JSON field type and we have many fields, it would be interesting to select or deselect them all.
![Captura de Tela 2021-08-19 às 17 48 04](https://user-images.githubusercontent.com/379850/130141822-36ac9e78-5796-475c-a185-fcfc0ae35aa8.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
